From fc73c01726652f47df53422d3cd816ac231e2f4e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pasi=20Sj=C3=B6holm?= <pasi.sjoholm@jollamobile.com>
Date: Wed, 11 Feb 2015 12:14:20 +0200
Subject: [PATCH 06/11] network: Make sure that IPv6 is enabled with
 autoconfiguration

Previously IPv6 enabled with manual_ipv6_net but not with
autoconfiguration.
---
 src/network.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/network.c b/src/network.c
index 0e979c9..217be32 100644
--- a/src/network.c
+++ b/src/network.c
@@ -614,6 +614,8 @@ static void autoconf_ipv6_set(struct connman_network *network)
 	if (!ipconfig)
 		return;
 
+	__connman_ipconfig_enable_ipv6(ipconfig);
+
 	__connman_ipconfig_address_remove(ipconfig);
 
 	index = __connman_ipconfig_get_index(ipconfig);
-- 
1.7.10.4

